applications/nrf5340_audio/src/nrf5340_audio_common.c

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
This seems a bit lost just in src, wouldn't it be better to have it in either modules or a common directory. I assume we will have an nrf54... file at some point? However as this is in nrf5340_audio application does it need the nrf5340_ prefix (similarly for audio_dk files).
How will this structure/code move/support nrf54?

Reply:
Good point, at some point we would have to rename this to fit both platform.